src/test/java/org/openmrs/contrib/qaframework/automation/EncountersSteps.java

```diff
@@ -0,0 +1,101 @@
+package org.openmrs.contrib.qaframework.automation;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.reference.page.AdmitToInpatientPage;
+import org.openmrs.reference.page.PatientVisitsDashboardPage;
+import org.openmrs.uitestframework.test.TestData;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+public class EncountersSteps extends Steps {
+
+	private TestData.PatientInfo testPatient;
+
+	@Before(RunTest.HOOK.SELENIUM_ENCOUNTER)
+	public void visitHomePage() {
+		testPatient = createTestPatient();
+		initiateWithLogin();
+	}
+
+	@After(RunTest.HOOK.SELENIUM_ENCOUNTER)
+	public void destroy() {
+		deletePatient(testPatient);
+		quit();
+	}
+
+	@Given("a user clicks on Find Patient app from Home page")
+	public void goToFindPatientRecordPage() {
+		findPatientPage = homePage.goToFindPatientRecord();
+	}
+
+	@Then("the system loads find patient record page")
+	public void systemloadsFindPatientRecordPage() {
+		assertTrue(textExists("Find Patient Record"));
+	}
+
+	@When("a user searches an existing patient and starts the visit")
+	public void searchExistingPatient() {
+		findPatientPage.enterPatient(testPatient.identifier);
+		findPatientPage.waitForPageToLoad();
+		visitsDashboardPage = findPatientPage.clickOnFirstPatient()
+				.startVisit();
+	}
+
+	@Then("the system loads patient visit dashboard page")
+	public void systemloadsVisitDashboardPage() {
```

Comment:
Is it visitDashboardPage or visitPage.

Reply:
The method loads patient visit dashboard page

Reply:
> The method loads patient visit dashboard page

Very true, thanks for clarification , have confirmed it[ here]([url](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/master/ui-tests/src/test/java/org/openmrs/reference/page/PatientVisitsDashboardPage.java)). how about calling it exact name , **LoadPatientVisitDashboardPage()**